test/checks/keyboard/landmark-is-top-level.js

```diff
@@ -45,26 +52,27 @@ describe('main-is-top-level', function () {
 		navDiv.setAttribute('role','navigation');
 		fixture.appendChild(navDiv);
 		fixture.appendChild(mainDiv);
-		assert.isTrue(checks['main-is-top-level'].evaluate(mainDiv));
+		assert.isTrue(check.evaluate.call(checkContext, mainDiv));
+		assert.deepEqual(checkContext._data, { role: 'main' });
 	});
 	
 	it('should return true if main is in form landmark', function () {
-		var mainDiv = document.createElement('div');
-		mainDiv.setAttribute('role','main');
+		var bannerDiv = document.createElement('div');
+		bannerDiv.setAttribute('role','banner');
```

Comment:
It feels like we're asking these existing tests to do double duty by adding in assertions that aren't exactly described by the test case. It might be better to break these out.